living_documentation_generator/github_projects.py

```diff
@@ -65,91 +90,115 @@ def __send_graphql_query(self, query: str) -> Optional[dict[str, dict]]:
         return None
 
     def get_repository_projects(self, repository: Repository, projects_title_filter: list[str]) -> list[GithubProject]:
+        """
+        Fetch all projects attached to a given repository using a GraphQL query. Based on the response create
+        GitHub project objects and return them in a list.
```

Comment:
objects or instances?

Reply:
Changed into `instances`, you are right!